_components/range-slider/range-slider.md

```diff
@@ -14,6 +14,14 @@ type: component
 maturity: alpha
 id_custom: range
 lead: A range slider allows users to choose an approximate number from a range.
+implementation:
+  initProps:
+    - property: "`data-text-unit`"
+      description: "Adds a unit to screen reader callouts. The unit will be read after the current value."
+      default: "`null`"
```

Comment:
Wondering if users might think it will read out "null", like "3.5 null of 5". We don't seem to use "null" as a value in our documentation. Suggest we either write the default as something like an empty value, or add a note in the description about what a default readout would be. 
```suggestion
      default: ""
```

Reply:
Leaving it empty almost looks _unintentional_  to me. I poked around and couldn't find any empty fields like this or alternative solutions. Here are a few screenshots I took while experimenting with other options. How do you feel about these?

![Screenshot 2023-11-01 at 3 45 16 PM](https://github.com/uswds/uswds-site/assets/25211650/83c5b6cd-4aa2-4daa-95e9-f43b74d7a9f2)
![Screenshot 2023-11-01 at 3 45 29 PM](https://github.com/uswds/uswds-site/assets/25211650/a3543a73-0a09-4ed6-96c6-2f1acc327cd9)
![Screenshot 2023-11-01 at 3 46 51 PM](https://github.com/uswds/uswds-site/assets/25211650/4d61a163-7d59-4d9c-9446-8414ce33ac6e)
![Screenshot 2023-11-01 at 3 46 13 PM](https://github.com/uswds/uswds-site/assets/25211650/e33c154a-a28c-4ea1-83a9-9ee1a07f8ce1)

Reply:
I like "Empty by default" or "undefined by default"